docs/release-process.md

```diff
@@ -1,8 +1,8 @@
-# Ansible Release Process
+# Manual Ansible Release Process
 
 ## Preamble
 
-This document describes the ansible community package release process.
+This document describes the "manual" ansible community package release process.
```

Comment:
I'd add a link here to the automated docs.

Reply:
Also done in 5ad0ad71749cb2907b8cdd6115246bbd5d9b8045.